.bash_prompt

```diff
@@ -103,14 +103,14 @@ else
 	hostStyle="${yellow}";
 fi;
 
-# Set the terminal title to the current working directory.
-PS1="\033]0; ${PWD##*/}\007";
+# Set the terminal title to the current working folder.
```

Comment:
`directory` and `folder` should mean the same thing.
I think you wanna call it `base path`. The opposite is `full path`.